frontend/src/pages/WorkspaceTeam/components/AutoJoinForm.tsx

```diff
@@ -1,107 +1,22 @@
-import { useAuthContext } from '@authentication/AuthContext'
-import { toast } from '@components/Toaster'
-import Tooltip from '@components/Tooltip/Tooltip'
-import {
-	useGetWorkspaceAdminsQuery,
-	useUpdateAllowedEmailOriginsMutation,
-} from '@graph/hooks'
-import { namedOperations } from '@graph/operations'
-import { Box, Select, Text } from '@highlight-run/ui/components'
-import { useParams } from '@util/react-router/useParams'
-import Checkbox, { CheckboxChangeEvent } from 'antd/es/checkbox'
-import React, { useState } from 'react'
-
-import { getEmailDomain } from '@/util/email'
+import Checkbox from 'antd/es/checkbox'
```

Comment:
we should replace this with one of our components  (to remove `antd` packages from dependencies)